packages/edit-post/src/components/visual-editor/index.native.js

```diff
@@ -26,6 +26,13 @@ class VisualEditor extends Component {
 		};
 	}
 
+	static getDerivedStateFromProps( props, state ) {
+		if ( props.isAnyBlockSelected && state.isPostTitleSelected ) {
```

Comment:
I wonder, can we remove the `&& state.isPostTitleSelected`? Like, as the PR description says anyway, the Post title and other blocks selection is mutually exclusive, right? If any block is selected, `isPostTitleSelected` needs to be false regardless of its previous state, no? I might be missing some context here though so, please feel free to elaborate. Thanks!

Reply:
Good idea, I've made this change. 

I believe I had that check just to avoid any possible risk of unnecessary re-renders, but looking at it again I think you're right and it's not needed. I can't find any documentation saying that kind of check is necessary, and I did some testing to confirm that removing the `&& state.isPostTitleSelected` part of the check does not cause any additional renders. Thanks @hypest !